client/src/components/Login.tsx

```diff
@@ -19,28 +19,49 @@ function usePasswordVisibility(){
 
   return(
     <>
-      <input type={passwordVisible ? "text" : "password"} placeholder="Password" />
+      <input type={passwordVisible ? "text" : "password"} placeholder="Password" 
+         onChange={(e: React.ChangeEvent<HTMLInputElement>)=> {
+           setPasswordLogin(e.target.value);
+         }}        
+       />
       <i onClick={togglePasswordVisibility}>{passwordVisible ? eye : eyeSlash}</i>
     </>
   );
 }
 
-
-
 export const Login: FC<Users> = (props: Users) => {
+  const [usernameLogin, setUsernameLogin] = useState<string>("");
+  const [passwordLogin, setPasswordLogin] = useState<string>("");
+  function loginButtonClick () {
+    console.log("You clicked me!");
+    console.log(usernameLogin);
+    console.log(passwordLogin);
+    
+    axios.post("http://localhost:3000/login", {
```

Comment:
`axios` is not defined anywhere in the component.

`npm install axios`

Add on line 2 `import axios from 'axios';`